annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/method/MethodRecord.java

```diff
@@ -40,8 +41,8 @@ public class MethodRecord {
   public Integer id;
   /** Location of the containing method. */
   public OnMethod location;
-  /** Is true if the method is already annotated with {@code Nullable} annotation. */
-  public boolean hasNullableAnnotation;
+  /** Set of annotations on method return type */
+  public String[] annotations;
```

Comment:
Why is this an array if the original thing that gets serialized is a set? Likely not a big deal, but I was wondering if there was a reason not to use `ImmutableSet` here too.

Reply:
That was just for simplicity since annotations are deserialized from a `tsv` column and were separated by calling `.split()` that returns an array. But it still worths making it immutable. dc785139139b843181938dab455e307fc8bcba1d